chatbot_ner/config.py

```diff
@@ -39,6 +40,35 @@
 ner_logger.addHandler(handler)
 ner_logger.addHandler(handler_stdout)
 
+# HAPTIK Environment and CAS name
+CLIENT_APPLICATIONS_SETUP_NAME = os.environ.get('CLIENT_APPLICATIONS_SETUP_NAME')
+HAPTIK_ENV = os.environ.get('HAPTIK_ENV')
```

Comment:
We should probably not introduce haptik specific stuff because this is open source

should call this `ENVIRONMENT` to keep it free of haptik terms
And prefer reading `ENVIRONMENT` first
```python
ENVIRONMENT = os.environ.get('ENVIRONMENT') or os.environ.get('HAPTIK_ENV')
```